examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -11103,7 +11088,7 @@
           "mfgCode": null,
```

Comment:
Turning off the tv clusters here will break tests that expect all-clusters-app to actually implement all clusters.  Why is this being done?

Reply:
Turned back on.